content-src/lib/selectLayoutRender.js

```diff
@@ -67,6 +67,15 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
   }
 
   const placeholderComponent = component => {
+    if (!component.feed) {
```

Comment:
we should add DSTextPromo to DS_COMPONENTS on L49 above so that it can be part of filterArray